system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.hpp

```diff
@@ -32,8 +34,9 @@ namespace system_metrics_collector
 class PeriodicMeasurementNode : public system_metrics_collector::Collector, public rclcpp::Node
 {
 public:
-  static constexpr const std::chrono::milliseconds DEFAULT_PUBLISH_WINDOW =
+  static constexpr const std::chrono::milliseconds INVALID_PUBLISH_WINDOW =
     std::chrono::milliseconds(0);
```

Comment:
TODO: please open ticket, we should **never** rely on 0 as a special value. std::optional or some other arbitrary constant number is preferable.

Reply:
Opened ticket https://github.com/ros-tooling/system_metrics_collector/issues/20, but there has been push back.

Reply:
This sentinel value is no longer of use in this program.